### PR TITLE
Google Play Store Release Blogpost

### LINKED ISSUE #222: Prepare Release-blog post Android Gplay + related project info
collect images and news here already: 

- new UX features: profile-images, the device-chat and saved messages, picture:  https://user-images.githubusercontent.com/9800740/70670840-87505c80-1c7a-11ea-8f5c-2ff78338c9a6.jpg

- Android9 is better supported (notifications, 64bit) 

- the core library of deltachat is now completely written in Safe Rust (multi-month effort, started April 2019), and is using modern async libraries to perform IMAP and SMTP. It will still require stabilization but already it's running faster. Delta Chat is thus the first messenger app suite that has a full-Rust core.   

- RPGP, the e2e crypto engine used by all DC apps, got a security review (link to publishing the security report), no critical flaws found for DC's usage, and most (all?) other issues have been addressed. 

- note that F-droid is not yet available, we aim for it once the gplay release stabilizes. Unfortunately, the build-process and feedback loop speed of releasing on F-droid leaves much to be desired so we couldn't tackle it in conjunction with the current release.  You can install an apk directly from our website though.  
<add more news/infos here, also images/screenshots and pictures> 

Desktop and iOS could probably better reported in a separate blog post.

### PR DESCRIPTION
**Pull request by ghost**
closes #222 

@hpk42 you had some other points?

### CODE REVIEW

**Comment by ghost** [comment]
Preview link: https://staging.delta.chat/225/en/2019-12-18-google-play-store-release

**Comment by r10s** [comment]
great writeup :)

wrt f-droid: i am about to file an issue on gitlab soon™, maybe we can add this, but i think this should not block things

File: _posts/2019-12-18-google-play-store-release.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+message them or when you "setup contact" with them through a QR-code. 
+
+As with all other core features of Delta Chat, there is no
+extra server (vulgo: "the cloud") involved, other than your e-mail
+provider for sending and receiving messages.  Note that all avatars are
+reduced to 192x192 pixels to spare everyone's bandwidth. 
+
+For privacy reasons, no one sees your profile picture until you write a message
+to them. This way, you can control who sees your picture without any extra
+interaction. This is very typical for the Delta approach on privacy by design.
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
interaction. This is very typical for the Delta approach of privacy by design.
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+<img src="../assets/blog/screenshots/2019-12-17-delta-chat-google-play-release-settings-light.png" width="180" 
+style="float: left; margin: 10px;display: block;box-shadow: 5px 5px 2px #777;" /> 
+
+One thing which makes Delta Chat much prettier are **profile pictures**. Or at
+least, it makes Delta as pretty as its users ;) You can now set an
+avatar image in your settings, and your contacts will see it when you
+message them or when you "setup contact" with them through a QR-code. 
+
+As with all other core features of Delta Chat, there is no
+extra server (vulgo: "the cloud") involved, other than your e-mail
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
extra server (vulgo: "the cloud") involved, other than your email
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+---
+title: Delta Chat 1.0 with new features, more robust, faster and more secure 
+author: compl4xx, holga
+---
+
+After months of hard work, we just released [Delta Chat 1.0 to Google Play store](https://play.google.com/store/apps/details?id=chat.delta), with lots of improvements. Even though a lot has changed under the hood we hope that everyone has a good upgrade experience.  If not, please submit a new [Android issue](https://github.com/deltachat/deltachat-android/issues) or visit the [support forum](https://support.delta.chat). <iframe width="370" height="208" src="https://www.youtube.com/embed/yPEjYpE_kvc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="float: right; margin: 10px;display: block;" ></iframe>
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
After months of hard work, we just released [Delta Chat 1.0 on the Google Play store](https://play.google.com/store/apps/details?id=chat.delta), with lots of improvements. Even though a lot has changed under the hood we hope that everyone has a good upgrade experience.  If not, please submit a new [Android issue](https://github.com/deltachat/deltachat-android/issues) or visit the [support forum](https://support.delta.chat). <iframe width="370" height="208" src="https://www.youtube.com/embed/yPEjYpE_kvc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="float: right; margin: 10px;display: block;" ></iframe>
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+
+The core system used by all Delta Chat apps is now written
+using the "Rust" language, widely hailed as the most robust and safe 
+system programming language. This also has speed up message receiving
+and delivery. Also, Android 9 is better supported regarding notifications 
+and 64bit support.
+
+For end-to-end encryption we are using
+[rPGP](https://github.com/rpgp/rpgp), which passed a security review
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
[rPGP](https://github.com/rpgp/rpgp), which passed a security
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+using the native system libraries on each platform. For protection
+against active attacks we have implemented and refined the [CounterMITM
+protocols](https://countermitm.readthedocs.io/en/latest/new.html) which
+in turn use rPGP. 
+
+## F-Droid coming later, needs your help! 
+
+F-Droid is still offering version 0.510.1 for now. We'll get to it 
+eventually but if someone wants to help along, please appear 
+in our [Contribution channels](https://delta.chat/en/contribute).
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
in our [contribution channels](https://delta.chat/en/contribute).
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+## Thanks to the dozens of people who helped! 
+
+The whole Rust-rewrite and this release was a massive community effort.  
+Thanks to all contributors, testers, translators, those who fetched mate & food
+and/or cooked at the gatherings, reviewed pull requests, thanks to our
+CI servers, and everyone else we forgot here.
+
+Oh, and if you don't have an Android phone, no problem - Delta Chat is available
+in preview versions for Windows, MacOS, Linux, and iOS. Those versions
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md:102]:
```suggestion
in preview versions for Windows, macOS, Linux, and iOS. Those versions 
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+## Thanks to the dozens of people who helped! 
+
+The whole Rust-rewrite and this release was a massive community effort.  
+Thanks to all contributors, testers, translators, those who fetched mate & food
+and/or cooked at the gatherings, reviewed pull requests, thanks to our
+CI servers, and everyone else we forgot here.
+
+Oh, and if you don't have an Android phone, no problem - Delta Chat is available
+in preview versions for Windows, MacOS, Linux, and iOS. Those versions 
+are all in Beta stage but are pretty well usable already:
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
are all in Beta stage but are pretty usable already: 
```

File: _posts/2019-12-18-google-play-store-release.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+review earlier in 2019. For Transport Layer Encryption (TLS) we are
+using the native system libraries on each platform. For protection
+against active attacks we have implemented and refined the [CounterMITM
+protocols](https://countermitm.readthedocs.io/en/latest/new.html) which
+in turn use rPGP. 
+
+## F-Droid coming later, needs your help! 
+
+F-Droid is still offering version 0.510.1 for now. We'll get to it 
+eventually but if someone wants to help along, please appear
```
Comment (comradekingu) [inline _posts/2019-12-18-google-play-store-release.md]:
```suggestion
eventually but if someone wants to help the process along, please appear 
```

**Comment by comradekingu** [comment]
Could use a few more commas and less "we".

File: _posts/2019-12-18-google-play-store-release.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+Previous to 1.0 Delta Chat showed pop-ups for warnings, like the infamous
+one about battery optimizations.  We are moving all such messages to the
+device chat so you can more easily scroll back. Speaking of battery optimizations --
+Delta Chat is pretty efficient with using your system resources and it is 
+usually fine to allow it to run in the background. However, as disscussed 
+in [Don't kill my app](https://dontkillmyapp.com/) Google doesn't always let us. 
+
+### Better Robustness, Speed and Security
+
+<img src="../assets/blog/rust-delta.png" width="180" style="float: left; margin: 10px;display: block;box-shadow: 5px 5px 2px #777;" />
```
Comment (Simon-Laux) [inline _posts/2019-12-18-google-play-store-release.md]: please remove the box shadow here, it looks odd

File: _posts/2019-12-18-google-play-store-release.md [hunk 10]
```diff
@@ -0,0 +1,102 @@
+---
+title: Delta Chat 1.0 with new features, more robust, faster and more secure 
+author: compl4xx, holga
+---
+
+After months of hard work, we just released [Delta Chat 1.0 to Google Play store](https://play.google.com/store/apps/details?id=chat.delta), with lots of improvements. Even though a lot has changed under the hood we hope that everyone has a good upgrade experience.  If not, please submit a new [Android issue](https://github.com/deltachat/deltachat-android/issues) or visit the [support forum](https://support.delta.chat). 
+
+<iframe width="370" height="208" src="https://www.youtube.com/embed/yPEjYpE_kvc" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="float: right; margin: 10px;display: block;" ></iframe>
```
Comment (Simon-Laux) [inline _posts/2019-12-18-google-play-store-release.md]: Please use youtube no-cookie or even better use only the thumbnail and clicking it brings the user to the video.

**Comment by ghost** [comment]
I made some wild changes to the page's css, so the images don't interfere with text readability on mobile. I also managed to embed the video in a way where it's always displayed nicely. I am happy with the layout so far, but there is still a checklist I want to look at before it can go out.

@hpk42 and I agreed on IRC that we will only publish this blogpost as soon as 1.0.1 is released. I'd wait with the checklist until then, in case some other things happen.

**Pull request merged**